packages/modals/src/elements/TooltipDialog/TooltipDialog.tsx

```diff
@@ -75,10 +69,13 @@ const TooltipDialogComponent = React.forwardRef<HTMLDivElement, ITooltipDialogPr
     const transitionRef = useRef<HTMLDivElement>(null);
     const [floatingElement, setFloatingElement] = useState<HTMLDivElement | null>();
     const [hasTitle, setHasTitle] = useState<boolean>(false);
+    const handleClose: ITooltipDialogProps['onClose'] = event => {
+      referenceElement && onClose?.(event);
```

Comment:
Looks like this is the important line. Checking for the presence of `referenceElement` before executing `onClose` is what will prevent the callback from being called twice. Is that right?